lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -4,6 +4,15 @@ define((require, exports, module) ->
   Range = require("ace/range").Range
   findSurroundingBrackets = require("ace/ext/papeeria/highlighter").findSurroundingBrackets
 
+  trim = (s) ->
```

Comment:
Is anything wrong with [String.trim()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim) ?